### ISSUE TITLE
 syncronization fails - Thunderbird freezes and won't start anymore

### ISSUE THREAD

**Issue by ichbk** [jobisoft/TbSync#218]
## Your environment

TbSync version: v0.7.22
Thunderbird version:60. Cannot check it anymore fr the exact version number
OS: Ubuntu 18.0.1 LTS

[x ] Yes, I have installed the latest available (beta) version from 
https://github.com/jobisoft/TbSync/releases 
and my issue is not yet fixed, I can still reproduce it.

## Expected behavior
(Office 365)
Everythings seems to work until i want to start the first sync.
I choose 15min for periodic sync.
Start sync, and expect  syncing to work.

## Actual behavior
it stucks at tasks. After that, pop-ups about not working scripts occur. i can provide screenshots.
Thunderbird then freezes and crashes.
Thunderbird does not start anymore, so i do not now how to provide a bug report.
I have tried a lot of delete and reinstall workarounds, but now thunderbird can not start.
Software center does not show me an remove possibility for thunderbird.

i had this problem months ago, but then i decided to stuck with windows.
Now on Ubuntu again and it would be beautiful, if we can find a solution for my problem.

## Steps to reproduce
First connects succesfully with outlook.com
First syncronisation fails badly.
Thunderbird does not even start anymore.

To help resolving your issue, enable debug logging (TbSync Account Manager -> Help) and send me the debug.log via e-mail (use the title of your issue as subject of the email).

**Comment by jobisoft** [comment]
screenshoot and some sort of error message would be good. Do you have any Special storage Driver? encrypted container? just Brainstorming.

You had the same issue on Windows?

You can start TB without addons (Google for that)

**Comment by jobisoft** [comment]
are you using the correct Version of Lightning? the one provided by the repo of your Distribution?

**Comment by ichbk** [comment]
> screenshoot and some sort of error message would be good. Do you have any Special storage Driver? encrypted container? just Brainstorming.
---->fastest solution for now   https://we.tl/t-otExRaaqfA 
> 
> You had the same issue on Windows?
-------> Yes i had
> 
> You can start TB without addons (Google for that)
-----> I could reinstall thunderbird via terminal and now i can start it again

**Comment by ichbk** [comment]
> are you using the correct Version of Lightning? the one provided by the repo of your Distribution?
---> Indeed i had pre-installed thunderbird 60 and i found a workaround for installing the newest lightning version.

i remember know, that the problem on windoes was similiar. When i removed lightning, at least thunderbird started again.

**Comment by ichbk** [comment]
I managed to send you a bug report with more Screenshots.

When i use "sudo apt install xul-ext-lightning", is that not the official repository?

**Comment by jobisoft** [comment]
Yes and Thunderbird must come from the same source, than they should match.

**Comment by ichbk** [comment]
Unfortunately, it did not help:
 
1. sudo apt-get remove thunderbird
2. deleted .thunderbird data @home directory
3. sudo apt-get install thunderbird
4. sudo apt install xul-ext-lightning
============================================
Thunderbird:60.2.1 (64-bit) 	
-> working
Lightning 6.2.2.1 		       
 -> working
============================================
After initial setup, everything works fine.
============================================
5. Install with Add-On Manager: Provider for Exchange ActiveSync
6. Install with Add-On Manager: TbSync
============================================
7. Succesfull restart of Thunderbird
8. Connecting with Outlook-Account:	
-> Success
9. Starting Sync:			                
-> Failure
-> tasks synced until [50/204] then TB chrashes
============================================

Any other ideas maybe?
